CHANGELOG.md

```diff
@@ -1,7 +1,79 @@
+# v0.2.0 - 2020-06-30
+* Adds the value transfer dApp:
+    * Support for transactions with Ed25519 and BLS signatures
+    * Parallel reality based ledger
+    * Support for colored tokens
+    * Conflict resolution via FPC
+    * Applied FCoB rules
+* Adds the network delay dApp which is used to gather the avg. network delay occurring in the network
+* Adds the faucet dApp giving the ability to request funds for testing via the dashboard or web API
+* Adds the DRNG dApp which is used to propagate random numbers produced by a dRand committee (this dApp is inactive)
+* Adds the base communication layer
+* Adds improved analysis server:
+    * Splits it into 3 separate plugins `analysis-client`/`analysis-dashboard`/`analysis-server`
+    * Uses TLV denoted messages for communication
+    * Complete new dashboard with live visualisations of the network graph and ongoing conflicts.
+    * Prometheus exporter to expose global network metrics
+    * Storage for conflicts inside a MongoDB for further processing
+    * Complete rewritten autopeering data retention
+* Adds additional HTTP API routes:
+    * "healtz" route to query the health of the node (for LBs)
+    * Query transaction attachment locations
+    * Query transactions by IDs
+    * Send transactions
+    * Get UTXOs on addresses
+    * Query info about the node
+    * Issue a faucet funding request
+    * Query dRNG committee
+    * Query dRNG random number
+    * Issue data payloads
+* Adds dashboard improvements:
+    * Tips chart
+    * Communication layer visualizer
+    * Address and UTXOs view
+    * Message payload view
+    * DRNG live feed
```

Comment:
* faucet page to request funds
* support different payload views:
  * data
  * value
  * faucet
  * unknown